### DISCUSSION TITLE
Change clean icon of input file and using `clearButtonProps`

### DISCUSSION THREAD

**Discussion by shahryarjb** [mantinedev/mantine#5327] [Q&A]
Hi, I have problem to use `clearButtonProps` in my `FileInput`

Before I show my code, I want to use it because I need to change clean or remove icon of `FileInput`

I tested it in diffrent shape
```javascript
<div className="mt-3">
      <FileInput
        leftSection={icon}
        label=""
        placeholder="آپلود فایل"
        dir="rtl"
        leftSectionPointerEvents="none"
        clearable
        clearButtonProps={
          <MyButton onClick={() => console.log('Button clicked')} type="button">
            Click me
          </MyButton>
        }
      />
```

and the component `MyButton`
```javascript
  const MyButton: React.FC<React.ComponentPropsWithoutRef<'button'>> = ({
    children,
    ...rest
  }) => {
    return <button {...rest}>{children}</button>;
  };

// or

const MyButton: React.FC<React.ComponentPropsWithoutRef<'button'>> = (props) => {
  return <button {...props} />;
};
```

Thank you in advanced

**Comment by rtivital** [comment]
`<FileInput rightSection={<YourButtonHere />} />`

**Comment by shahryarjb** [comment]
> `<FileInput rightSection={<YourButtonHere />} />`

The button I added is always run the open file input not the command I expect, i did z-index and stop prevent default and propgate but it does not work for me
@rtivital

**Reply by rtivital** [reply]
Set `rightSectionPointerEvents="all"`
